src/components/ContactForm/index.tsx

```diff
@@ -58,6 +58,15 @@ const Contact = ({ title, content, id, t }: ContactProps) => {
                 />

```

Comment:
- **Pre-condition check**  
It seems that the `values.contact` is used without checking if it exists. It would be better to add a null check to prevent runtime errors.
- **Runtime error check**  
There is a potential runtime error if `values` is null or undefined. It should be checked before accessing its properties.